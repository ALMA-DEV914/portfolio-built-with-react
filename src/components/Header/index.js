import React from 'react';
import Nav from "../Nav";

function Header(props) {
  const { currentCategory, setCurrentCategory } = props;

  return (
    <header className='flex-row px-1'>
      <div>
        <h1 className='header' >Alma Braun</h1>
      </div>
      <div>
        <Nav
          currentCategory={currentCategory}
          setCurrentCategory={setCurrentCategory}
        ></Nav>
      </div>
    </header>
  );
}

export default Header;