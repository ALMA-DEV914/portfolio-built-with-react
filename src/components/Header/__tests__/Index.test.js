import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '..';

afterEach(cleanup);

describe('About component', () => {
    //renders About test
    it('renders', () => {
        render(<Header />);
    });
    it('matches snapshot DOM node structure', () => {
        const {asFragment} = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
    });
})
