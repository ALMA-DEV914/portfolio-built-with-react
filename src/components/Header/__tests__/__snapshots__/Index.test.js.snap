// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <header
    class="flex-row px-2"
  >
    <div>
      <h1
        class="header"
      >
        Alma Braun
      </h1>
    </div>
    <div
      class="nav"
    >
      <nav>
        <ul
          class="flex-row"
        >
          <li
            class="mx-2"
          >
            <span>
              About Me
            </span>
          </li>
          <li
            class="mx-2"
          >
            <span>
              Portfolio
            </span>
          </li>
          <li
            class="mx-2"
          >
            <span>
              Contact
            </span>
          </li>
          <li
            class="mx-2"
          >
            <span>
              Resume
            </span>
          </li>
        </ul>
      </nav>
    </div>
  </header>
</DocumentFragment>
`;
